configs/next/packages/libnxz/stage_1

```diff
@@ -37,11 +37,6 @@ atcfg_pre_configure() {
 atcfg_configure() {
 	# libnxz do not use autoconf
 	${SUB_MAKE} clean
-	# Hack to get the version outside the git repository.
-	cat lib/Makefile | \
-		sed "s/git describe --tags | cut -d - -f 1,2 | tr - ./echo ${ATSRC_PACKAGE_VER}/" \
-		> lib/Makefile.temp
-        [[ ${?} -eq 0 ]] && mv lib/Makefile.temp lib/Makefile || exit 1
```

Comment:
We will still need this hack because the git repository will be on `sources` and the build is done on `$AT_WD/sources`, that won't have the `.git` directory and files.

Reply:
@rzinsly @mscastanho My understanding is this hack is needed when we take the sources from the tarball as AT <= 15.0 does. But AT next will `git clone` the sources, so `git` commands will be available and then the hack won't be necessary.

Reply:
@er-1 `lib/Makefile` on libnxz uses `git describe --tags` on line 6, that command won't be available at build time because the source copied to `$AT_WD/sources` is not a git repository, you only are able to run that in the `sources` at the top directory, there is where the `git clone` is done.

Reply:
> `lib/Makefile` on libnxz uses `git describe --tags` on line 6, that command won't be available at build time because the source copied to `$AT_WD/sources` is not a git repository

@rzinsly The `.git` directory isn't copied with the whole sources? I thought the copy was done using `cp -r`, so `.git`would be copied making the directory under `$AT_WD/sources` a git repository.

Reply:
@er-1 the copy is done with rsync and we explicit exclude all version control files with `--exclude=CVS --exclude=.svn --exclude=.git`

Reply:
Fixed.